### PR TITLE
Add FSU tech community assesment

### PR DESCRIPTION
**Pull request by Lionex**
Answers questions detailed in docs/community-assesment.md as directed in week 1 todo.

Fills out assessment from two perspectives:

- FSU graphics community
- Overarching FSU tech community

### CODE REVIEW

**Comment by hackerkid** [comment]
@Lionex I think you can probably squash all the 3 commits into 1.

**Comment by Lionex** [comment]
Don't know how to do that for a pull request.

**Comment by hackerkid** [comment]
@Lionex You have to use git rebase command. It's pretty easy. Check out [this](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html). Yeah they do have the option for squashing. From the maintainers point of view it's always good to see PRs with proper commits. :smile:

**Comment by Lionex** [comment]
@hackerkid I wasn't aware that a rebase on my end would appear here.  Will do!

**Comment by Lionex** [comment]
@hackerkid as I thought, it just created a conflict with the remote and I ended up adding more commits.  Any way to fix that now?

**Comment by hackerkid** [comment]
@Lionex I think you should have used `git push origin master --force-update` after the rebase. Now you can remove all the useless commits and do a force update. Force update overwrites the remote branch with the same copy as your local branch (it wont check for any merge conflicts). Always take extra care when you use force update as it replaces the remote branch completely with a local version.

**Comment by Lionex** [comment]
@hackerkid Thank you so much, it worked!

I will do this more often.

File: communities/fsu-tech/community-assessment.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
+## Assessing Your Community's Needs
+
+---
+
+### Definition
+
+I see two communities I would like to imporove here at FSU:
```
Comment (PeterDaveHello) [inline communities/fsu-tech/community-assessment.md]: `imporove` -> `improve`?

**Pull request merged**